helm/flink-operator/templates/flink-operator.yaml

```diff
@@ -123,6 +127,12 @@ spec:
                 path: flink-conf.yaml
               - key: log4j-console.properties
                 path: log4j-console.properties
+        {{- if .Values.hostPath.create }}
+        - name: {{ .Values.hostPath.name }}
+          hostPath:
+            path: {{ .Values.hostPath.hostPath }}
+            type: DirectoryOrCreate
+        {{- end }}
```

Comment:
This logic is minikube specific right? Maybe it would be better to allow the user to define a volume-mount directly. That way they can use hostPath / persistentvolumeclaim etc.

Reply:
What about provide two option here: hostPath and persistentvolumeclaim. Users can choose to mount the hostPath or persistentvolumeclaim or none ?